src/components/android-settings-container.js

```diff
@@ -174,16 +165,15 @@ class AndroidSettingsContainer extends React.Component {
                                           numberSmsConfirmation: '',
                                           valuesTEI: '',
                                           encryptDB: encryptDB,
+                                          isUpdated: true,
+                                          loading: false,
```

Comment:
This looks like a functional change, should it be in a different PR?

Reply:
I will remove it and create a new PR just for this change